tests/docs/running-perf-tests.md

```diff
@@ -144,3 +144,150 @@ make test-perf-all
 To keep the build infrastructure simple, Dapr uses dapr-test GitHub Actions Workflow to run e2e tests using one of AKS clusters. A separate workflow also runs E2E in KinD clusters.
 
 Once a contributor creates a pull request, E2E tests on KinD clusters are automatically executed for faster feedback. In order to run the E2E tests on AKS, ask a maintainer or approver to add /ok-to-perf comment to the Pull Request.
+
+
+
+
+# Setup Required For Visualising Performance Test Metrics
+
+The setup in AKS requires three servers for:
+ - Prometheus 
+ - Pushgateway
+ - Grafana
```

Comment:
Install the following in your Kubernetes cluster:

Reply:
Changed.